e2e-tests/cypress/e2e/journeys/insurance/your-business/broker/broker-page.spec.js

```diff
@@ -0,0 +1,110 @@
+import { broker } from '../../../../pages/your-business';
+import partials from '../../../../partials';
+import { submitButton, saveAndBackButton } from '../../../../pages/shared';
+import { PAGES, BUTTONS, LINKS } from '../../../../../../content-strings';
+import { ROUTES } from '../../../../../../constants';
+import { EXPORTER_BUSINESS as FIELD_IDS } from '../../../../../../constants/field-ids/insurance/exporter-business';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.EXPORTER_BUSINESS.BROKER;
+
+const {
+  BROKER: {
+    USING_BROKER,
+  },
+} = FIELD_IDS;
+
+const {
+  ROOT,
+  START,
+  EXPORTER_BUSINESS: {
+    BROKER,
+  },
+} = ROUTES.INSURANCE;
+
+const insuranceStart = ROUTES.INSURANCE.START;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.exporterBusiness;
+
+context('Insurance - Your business - Broker Page - As an Exporter I want to confirm that I am not using a broker for my export Insurance so that UKEF and I can easily collaborate and manage correspondence regarding my export insurance', () => {
```

Comment:
Since the test description says "...want to confirm I am not using a broker", should we have a test for form submission? e.g

```js
describe('form submission with all valid required fields', () => {
  it(`should redirect to ${ENTER_CODE}`, () => {
```

Reply:
Sorry just seen this is in the validation test, can we rename this description then to 

"...want to confirm if I am using ..."